common/include/pcl/point_cloud.h

```diff
@@ -227,6 +227,7 @@ namespace pcl
       {}
 
       //TODO: check if copy/move contructors/assignment operators are needed
+      //TODO: remove casting of the return value of size() to index_t
```

Comment:
I don't know if this would be picked up by doxygen

Reply:
I'll add it as a doxygen command.